examples/ci_cd/gitlab-ci-cd/README.md

```diff
@@ -1,29 +1,45 @@
-# Gitlab CI/CD example
-This pattern shows the example to deploy the Amazon EKS Clusters using Gitlab CI/CD
+# GitLab CI/CD example
+This pattern shows an example of how to deploy an Amazon EKS Cluster using GitLab CI/CD.
 
-### Step1: Clone this repo
+### Step 1: Clone this repo
 
 ```
 git@github.com:aws-samples/aws-eks-accelerator-for-terraform.git
 ```
 
-## Step2: Create a new Gitlab repo
-Copy this folder(`gitlab-ci-cd`) to your new gitlab repo. Rename the folder according to your naming convention.
+## Step 2: Create a new Gitlab repo
+Copy the files in the folder(`gitlab-ci-cd`) into the root of your new GitLab repo.
 
     cd examples/advanced/gitlab-ci-cd
-    gitlab-ci-cd
+    cp . $YOUR_GITLAB_REPO_ROOT
 
-## Step3: Update CI/CD settings config
- - Login to gitlab console, click on your repo and navigate to `settings/ci_cd`
+## Step 3: Update CI/CD settings config
+ - Login to the GitLab console, click on your repo and navigate to `settings/ci_cd`
  - Add the following variables before triggering the pipeline
 
         AWS_ACCESS_KEY_ID           e.g., access key from devops admin iam role
         AWS_SECRET_ACCESS_KEY       e.g., secret key from devops admin iam role
         AWS_REGION                  e.g., eu-west-1
-        GITLAB_BASE_URL             e.g., https://gitlab.aws.dev/api/v4/
-        GITLAB_TOKEN                e.g., gitlab access token  
+        GITLAB_BASE_URL             e.g., https://gitlab.example.com/api/v4/
+        GITLAB_TOKEN                e.g., GitLab access token 
 
-## Step5: Commit changes to verify the pipeline
+## Step 4: Commit changes to verify the pipeline
+
+Start the `tf-destroy` step in the GitLab Ci/CD pipeline to destroy your deployment.
+
+## Troubleshooting:
+
+- ### 403 Error when creating resource gitlab_group_cluster.aws_cluster
+
+    - Make sure the GitLab group that you're referencing in `data.gitlab_group.gitops-eks` exists and that you have access to it.
+
+- ### 400 Error when creating resource
+
+    - If the error contains `{message: {environment_scope: [cannot add duplicated environment scope]}}`, it is likely that an existing Kubernetes integration with the same environment scope was not removed. Remove any Kubernetes clusters with the same environment scope from the GitLab group before redeploying.
+
+- ### What's gitlab-terraform?
+
+    - `gitlab-terraform` is a thin wrapper around the `terraform` binary as part of the [GitLab Terraform docker image](registry.gitlab.com/gitlab-org/terraform-images/stable:latest) used in `.gitlab-ci.yml`.
```

Comment:
`registry.gitlab.com/gitlab-org/terraform-images/stable:latest` is image link, probably wanted to have doc link here?

Reply:
link updated